pageserver/src/tenant.rs

```diff
@@ -4064,7 +4050,8 @@ pub(crate) mod harness {
             info_span!("TenantHarness", tenant_id=%self.tenant_shard_id.tenant_id, shard_id=%self.tenant_shard_id.shard_slug())
         }
 
-        pub async fn load(&self) -> (Arc<Tenant>, RequestContext) {
+        #[cfg(test)]
```

Comment:
Darn, forgot this one. I must've been quite confused jumping between the preload paths as I thought that these were in prod code.